packages/manager/apps/metrics/src/index.html

```diff
@@ -3,7 +3,8 @@
     <head>
         <title>Metrics App</title>
     </head>
-    <body data-ng-app="metricsApp" data-ng-strict-di>
```

Comment:
Out of curiosity, why removing `data-ng-strict-di`?

Reply:
@antleblanc It is added when bootstraping the angular app https://github.com/ovh/manager/pull/4276/files#diff-4942f69008ca76ca4c4beb838787d8f15a85b4172bcdba5decc8961f44e793bfR15 not removed just moved 😉